### PR TITLE
Feature/order management

### PR DESCRIPTION
**Pull request by karnse**

### CODE REVIEW

File: src/pages/admin/order-management.tsx [hunk 1]
```diff
@@ -1,56 +1,105 @@
...
         </PageHeader>
         <OrderContainer>
           {allOrderStatus
             .filter(
-              (status) => status,
+              (status) => {
+                if (filterStatus.length === 0) return true;
+                return filterStatus.includes(status);
+              },
               // todo: filter order status by selected status
```
Comment (JiMeow) [inline src/pages/admin/order-management.tsx:58]: remove comment

File: src/pages/admin/order-management.tsx [hunk 2]
```diff
@@ -1,56 +1,105 @@
...
             )
             .map((status) => {
               // allOrder.filter((order) => order.status === status)
```
Comment (JiMeow) [inline src/pages/admin/order-management.tsx:61]: remove comment

File: src/modules/admin/order/components/PopOverFilter.tsx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+        .filter((status) => status !== "CANCELLED")
+        .map((status) => reverseStatusMarksTranslation[status] as number),
+    ),
+    Math.max(
+      ...filterStatus
+        .filter((status) => status !== "CANCELLED")
+        .map((status) => reverseStatusMarksTranslation[status] as number),
+    ),
+  ]);
+  console.log("changeFilterStatusSlider", changeFilterStatusSlider);
```
Comment (JiMeow) [inline src/modules/admin/order/components/PopOverFilter.tsx:66]: remove log?

File: src/modules/admin/order/components/PopOverFilter.tsx [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+          .filter((status) => status !== "CANCELLED")
+          .map((status) => reverseStatusMarksTranslation[status] as number),
+      ),
+      Math.max(
+        ...filterStatus
+          .filter((status) => status !== "CANCELLED")
+          .map((status) => reverseStatusMarksTranslation[status] as number),
+      ),
+    ]);
+    console.log("changefilterslider", changeFilterStatusSlider);
```
Comment (JiMeow) [inline src/modules/admin/order/components/PopOverFilter.tsx:132]: remove log?

**Review by JiMeow — COMMENTED** [review]
All hook look good, I will check components later

File: src/pages/admin/order-management.tsx (hunk 2 above)
Comment (karnse) [reply src/pages/admin/order-management.tsx:61]: done

File: src/pages/admin/order-management.tsx (hunk 1 above)
Comment (karnse) [reply src/pages/admin/order-management.tsx:58]: done

File: src/modules/admin/order/components/PopOverFilter.tsx (hunk 3 above)
Comment (karnse) [reply src/modules/admin/order/components/PopOverFilter.tsx:66]: done

File: src/modules/admin/order/components/PopOverFilter.tsx (hunk 4 above)
Comment (karnse) [reply src/modules/admin/order/components/PopOverFilter.tsx]: done

**Comment by GolfGrab** [comment]
@karnse 
pls change all action element from div to ant buttons
image

**Review by GolfGrab — CHANGES_REQUESTED** [review]
fix button

**Comment by GolfGrab** [comment]
CANCELLED reason that not related with ingredients and addon not show up 
image

image

**Comment by GolfGrab** [comment]
@karnse let me continue this pr for you

**Review by GolfGrab — APPROVED** [review]
nice work

**Pull request merged**